dockers/docker-orchagent/Dockerfile.j2

```diff
@@ -10,13 +10,17 @@ RUN apt-get update
 
 RUN apt-get install -f -y ifupdown arping libdbus-1-3 libdaemon0 libjansson4
 
-RUN apt-get install -f -y ndisc6
+RUN apt-get install -f -y ndisc6 tcpdump python-scapy
 ## Install redis-tools dependencies
 ## TODO: implicitly install dependencies
 RUN apt-get -y install libjemalloc1
 
 RUN apt-get install -y libelf1 libmnl0
 
+RUN pip install --upgrade pip
+RUN pip install setuptools 
```

Comment:
why do we need setuptools here?

Reply:
It is required to install "pyroute2 netifaces"